config-base.js

```diff
@@ -103,53 +142,44 @@ module.exports = {
       { blankLine: 'always', prev: '*', next: ['const', 'let', 'var'] },
       { blankLine: 'any', prev: ['const', 'let', 'var'], next: ['const', 'let', 'var'] },
     ],
-    'prefer-const': 'warn',
-    'prefer-template': 'warn',
-    // disable atomic updates rule until this issue is resolved: https://github.com/eslint/eslint/issues/11899
-    'require-atomic-updates': 'off',
-    'require-await': 'off', // use @typescript-eslint/no-floating-promises instead
```

Comment:
removed this but think it needs to be re enabled to allow for @typescript-eslint/no-floating-promises to take precedence